src/components/Dashboard/Form/Team/TeamMembers/TeamMembers.js

```diff
@@ -18,18 +18,8 @@ const TeamMembers = () => (
         </Button>
         {fields.map((element, index) => (
           <div key={index} className={styles.TeamMembersInputWrapper}>
-            <FieldWrapper
-              name={`${element}.name`}
-              component="input"
-              componentSize="medium"
-              label="Imię i nazwisko"
-            />
-            <FieldWrapper
-              name={`${element}.function`}
-              component="input"
-              componentSize="medium"
-              label="Funkcja"
-            />
+            <TextField name={`${element}.name`} />
+            <TextField name={`${element}.function`} />
```

Comment:
You can extend `TextField` component by adding `label` props. For now `TextField` does not support custom labels and without that on the last step of the form inputs will lose their labels. (Or just restore FieldWrapper in this place and label prop can be done in other ticket)